### ISSUE TITLE
creating a public hub world

### ISSUE THREAD

**Issue by charlie-sans** [Yellow-Dog-Man/Resonite-Issues#2928]
### Is your feature request related to a problem? Please describe.

when new people join resonite for the first time, 95% of the time they will find a session with enough people in it to learn how to play resonite.

sometimes though that might not be the case and it might draw away from the learning aspect of resonite.

### Describe the solution you'd like

having a "hub" world that allows users to hang out, chill and maybe even submit public items/folders like the resonite essentials folder and the old neos hub we used to have back then.

### Describe alternatives you've considered

getting together with a few friends and making a unofficial hub for resonite or using sonako's public folder's library 

### Additional Context

i'm not so sure how this would work, but i hope this might be a good idea.

### Requesters

charlie_san

**Comment by shiftyscales** [comment]
This was something I was hoping we could introduce with official headless sessions- my particular vision was to have additional teleport destinations at the end of the tutorial which allow users to hop into various experiences of their choosing depending on the kind of experience they're after, e.g. social, build, play, etc.

An initial version of that is in-progress, but currently blocked by #2905.

Item / object submission would require #1473 which is a separate and much larger issue.

**Comment by lxw404** [comment]
I think this is a huge missing part of the current culture here, which didn't persist from another place.  Having an official central location/world to meet up and know that people there might need help immediately coming from the tutorial would be extremely beneficial for fostering a culture of shared knowledge and helping, which mentors and regular users alike work towards.  However at the moment, this is extremely ad hoc, and even more intimidating: a user has to actively go and seek out help in public worlds that they've never been to before, and may find overwhelming.

**Comment by shiftyscales** [comment]
Yeah- that was kind of my hope @lxw404 - have neutral and known spaces which mentors, etc. can casually hang out in day to day, and if a new user happens to come on through via the tutorial, they'll have a space that fits whatever need they were after- otherwise if the user wants to take things at a slower pace, they could just continue to their cloud home as a private session like what happens currently.

**Comment by Lorgenz** [comment]
One thing that could be done in the meantime is to add some "community spotlighted" tutorials at the end of the current official one such as linking to many of the user created tutorial worlds and objects along with places like the new avatar station.
